.github/workflows/tests.yml

```diff
@@ -42,3 +33,17 @@ jobs:
 
       - run: yarn
       - run: yarn test
+
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
```

Comment:
Onboard to use Oliver's fork but what's the motivation/impetus?

Reply:
No, just curious. In my experience they do great work so it's probably better in some way just don't know what haha. They're one of the big open source contributors to the foundry ecosystem afaik.